tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -744,9 +746,19 @@ namespace Tensile
         }
     }
 
-    template <bool T_Debug, typename KA>
-    void ContractionSolution::kernelArgs(KA& args, const ContractionProblemParameters& param) const
+    template <bool T_Debug, bool Legacy, typename KA>
+    void ContractionSolution::kernelArgs(uint32_t                            gemmCount,
+                                         uint32_t                            argType,
```

Comment:
What is argType? should we use enum for different type?

Reply:
In stage 1 we only have 0 and 1. In stage 2 we'll have different types.

Reply:
Add comment in code